floss/language/go/extract.py

```diff
@@ -0,0 +1,410 @@
+            section_data = section.get_data(section_va, section_size)
+
+            if alignment == 0x10:
+                yield from chain(
+                    extract_longstrings64(
+                        pe, section_data, section_va, min_length, extract_longstring64, regex_offset=7
+                    ),
+                    extract_longstrings64(
+                        pe, section_data, section_va, min_length, extract_longstring64_2, regex_offset=13
+                    ),
+                    extract_longstrings64(
+                        pe, section_data, section_va, min_length, extract_longstring64_3, regex_offset=15
+                    ),
+                    extract_longstrings64(
+                        pe, section_data, section_va, min_length, extract_longstring64_4, regex_offset=20
+                    ),
+                )
+
+            else:
+                yield from chain(
+                    extract_longstrings32(pe, section_data, section_va, min_length, extract_longstring32),
+                    extract_longstrings32(pe, section_data, section_va, min_length, extract_longstring32_2),
+                    extract_longstrings32(pe, section_data, section_va, min_length, extract_longstring32_3),
+                )
+
+        if section_name == ".rdata":
+            section_va = section.VirtualAddress
+            section_size = section.SizeOfRawData
+            section_data = section.get_data(section_va, section_size)
+
+            yield from chain(
+                extract_string_blob(pe, section_data, section_va, min_length),
+                extract_string_blob2(pe, section_data, section_va, min_length),
+            )
+
+        if section_name == ".text":
+            # Extract string in .text section
+            section_va = section.VirtualAddress
+            section_size = section.SizeOfRawData
+            section_data = section.get_data(section_va, section_size)
+
+            yield from extract_build_id(pe, section_data, section_va)
+
+            for m in extract_stackstring_pattern.finditer(section_data):
+                for i in range(1, 8):
+                    try:
+                        tmp_string = m.group(i)
+                        if tmp_string != b"":
+                            try:
+                                decoded_string = tmp_string.decode("utf-8")
+                                if decoded_string.isprintable() and len(decoded_string) >= min_length:
+                                    addr = m.start() + pe.OPTIONAL_HEADER.ImageBase + section_va
+                                    yield StaticString(
+                                        string=decoded_string, offset=addr, encoding=StringEncoding.ASCII
+                                    )
+                            except UnicodeDecodeError:
+                                pass
+                    except AttributeError:
+                        pass
+
+        if section_name == ".idata":
```

Comment:
what if the imports are in a section named something else? with pefile we can see where the imports are stored